### PR TITLE
docs(admin_backend_doc_readme): create documentation for use backend …

### PR DESCRIPTION
**Pull request by PriscilliaAmmeux**
…admin

# Description
What are changes related to ? 
As a developer,
I want to update the readme,
In order to be able to understand how to launch the application. Understand what is the purpose of the application. Guide to other documentation files.

# Linked Issues
#355

# Changes
it is documentation

# Screenshots
Put screenshots (if any)

# Tests
How has it been tested ?

no

# Additional information
Precise any other information

### CODE REVIEW

File: yaki_admin_backend/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# First Utilisation:
+
+## application.properties:
+
+1 - The `application.properties` file provides a template for creating an environment configuration file. It contains a list of environment variables and their default values, which can be used as a starting point for creating a `application.properties` file.
+
+The `application.properties` file is used to store sensitive information such as API keys, database credentials, and other configuration settings that should not be hard-coded in the application code. We recommend that you copy `env.sample` to `application.properties.sample` and customize it with your own configuration settings
+
+2 - Open intelliJi and build clean and assemble.
+
```
Comment (profotoce59) [inline yaki_admin_backend/CONTRIBUTING.md:14]: we need to specify that to run the docker compose we need to fill the .env located in the yaki_docker folder

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ For more information about `Pull Request` creation please visit [https://docs.gi
 ## Naming Convention
 
 For this project, naming convention for all language use in this project should be on lower camelCase,
-except for Classes, enum types, typedefs, and type parameters these one should be on Upper CamelCase (PascalCase)
+except for Classes, enum types, typedefs, and type parameters these one should be on kebab-case.
```
Comment (SDecambray) [inline CONTRIBUTING.md:169]: Classes, enum type, typedefs, and type parameters should be on PascalCase

Components references in Yaki Admin (VueJS) should be on  kebab-case

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -176,7 +176,7 @@ For a new files : new_file_create.dart
 ```
 
 ```
-For a new Classes : class MyClasse
+For new class: title-presentation
```
Comment (SDecambray) [inline CONTRIBUTING.md:179]: class: TitlePresentation

components in the template in yaki-admin: title-presentation

File: yaki_admin/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+```
+For a new constant variable : const myNewConst = x
+```
+
+## env.sample
+
+The `env.sample` file provides a template for creating an environment configuration file. It contains a list of environment variables and their default values, which can be used as a starting point for creating a `.env` file.
+
+The `.env` file is used to store sensitive information such as API keys, database credentials, and other configuration settings that should not be hard-coded in the application code. We recommend that you copy `env.sample` to `.env` and customize it with your own configuration settings.
+
```
Comment (profotoce59) [inline yaki_admin/CONTRIBUTING.md:202]: application.properties.sample is not mentionned

File: yaki_backend/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+```
+For a new constant variable : const myNewConst = x
+```
+
+## env.sample
+
+The `env.sample` file provides a template for creating an environment configuration file. It contains a list of environment variables and their default values, which can be used as a starting point for creating a `.env` file.
+
+The `.env` file is used to store sensitive information such as API keys, database credentials, and other configuration settings that should not be hard-coded in the application code. We recommend that you copy `env.sample` to `.env` and customize it with your own configuration settings.
+
```
Comment (profotoce59) [inline yaki_backend/CONTRIBUTING.md:203]: application.properties.sample is not mentionned

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -222,3 +222,57 @@ divided into 2 main parts :
...
+
+The `env.sample` file provides a template for creating an environment configuration file. It contains a list of environment variables and their default values, which can be used as a starting point for creating a `.env` file.
+
+The `.env` file is used to store sensitive information such as API keys, database credentials, and other configuration settings that should not be hard-coded in the application code. We recommend that you copy `env.sample` to `.env` and customize it with your own configuration settings.
+
+# application.properties.sample
+
+The `application.properties` file provides a template for creating an environment configuration file. It contains a list of environment variables and their default values, which can be used as a starting point for creating a `application.properties` file.
+
+The `application.properties` file is used to store sensitive information such as API keys, database credentials, and other configuration settings that should not be hard-coded in the application code. We recommend that you copy `application.properties.sample` to `application.properties` and customize it with your own configuration settings.
```
Comment (SDecambray) [inline CONTRIBUTING.md:245]: application.properties do not store API key

File: yaki_admin/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+## Pull Requests
+
+Each change must be proposed to merge using a `Pull Request` from the current branch to the main one
+
+For more information about `Pull Request` creation please visit [https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+
+## Naming Convention
+
+For this project, naming convention for all language use in this project should be on lower camelCase,
+except for Classes, enum types, typedefs, and type parameters these one should be on kebab-case.
```
Comment (SDecambray) [inline yaki_admin/CONTRIBUTING.md:180]: Only component reference in the template should be on kebab-case everything else should be on PascalCase

File: yaki_admin_backend/README.md [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+  - saveOrUpdate => update a captain/customer/team/ teamMate by id
+
+- GetMapping({"captain/{id}"}):
+
+  - findAllByCaptain => find all team by captain id
+
+- GetMapping("team/{id}"):
+  - findAllByTeam(id) => find all teamMate by team id.
+
+<!-- add login path -->
```
Comment (SDecambray) [inline yaki_admin_backend/README.md:68]: comment ?

**Review by SDecambray — APPROVED** [review]

**Pull request merged**